command/node_pool_init.go

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package command
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"strings"
+
+	"github.com/posener/complete"
+)
+
+const (
+	// DefaultHclNodePoolInitName is the default name we use when initializing
+	// the example node pool spec file in HCL format
+	DefaultHclNodePoolInitName = "pool.np.hcl"
```

Comment:
```suggestion
	DefaultHclNodePoolInitName = "pool.nomad.hcl"
```

I think text editors would prefer if we were consistent with file extension? Wrapping things in blocks then provides a consistent grammar.

Reply:
Makes sense. We didn't do that for variables (or volumes, or quotas, or namespaces... _sigh_ :grinning: ) but if we can do it going forward it'll let us define that grammar as a single "nomad grammar"